nearcore/src/config_validate.rs

```diff
@@ -201,17 +201,13 @@ impl<'a> ConfigValidator<'a> {
 
 #[cfg(test)]
 mod tests {
-    use near_primitives::types::ShardId;
-
     use super::*;
 
     #[test]
     #[should_panic(expected = "gc config values should all be greater than 0")]
     fn test_gc_config_value_nonzero() {
         let mut config = Config::default();
         config.gc.gc_blocks_limit = 0;
-        // set tracked_shards to be non-empty
-        config.tracked_shards.push(ShardId::new(20));
```

Comment:
Do you need to set it to AllShards? same for other two occurrences in this file

Reply:
This was needed in the past, but config validation no longer checks `tracked_*` fields and passes without it.